service.go

```diff
@@ -38,11 +39,6 @@ type Service interface {
 type Config struct {
 	BasicClientConfig chrysom.BasicClientConfig
 
-	// Logger for this package.
-	// Gets passed to Argus config before initializing the client.
-	// (Optional). Defaults to a no op logger.
-	Logger *zap.Logger
-
```

Comment:
Utilizing context get/set loggers instead in order to simplify setup and keeping things consistent